src/technologies/index.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: article
+title: "Technologies"
```

Comment:
Worth mentioning databases and IDEs here?

Reply:
I've moved the related article on Data stores into this section as it was so small.
I've mentioned IDEs on the Java tech article. I'm waiting on Damian for a similar article on Node.
I think we can go further into IDEs when we've looked at how to encourage best practice in them